.github/workflows/ci.yaml

```diff
@@ -16,7 +12,6 @@ jobs:
   # test action works running from the graph
   test:
     name: Test
-    needs: [lint]
```

Comment:
@dgschwindturo @shakefu I couldn't think of any reason to have these run after `lint`?  Is there something that I missed?